core/src/main/scala/org/bitcoins/core/psbt/PSBT.scala

```diff
@@ -58,11 +68,211 @@ case class PSBT(
       vec.head.version
     }
   }
+
+  /**
+    * @param tx Transaction to add to PSBT
+    * @param i index of inputMap to add tx to
+    * @return PSBT with added tx
+    */
+  def addTransaction(tx: Transaction, i: Int): PSBT = {
```

Comment:
I think it would be nice to rename this to be more descriptive of what is happening here since I could mis-interpret this to mean messing with the underlying unsigned tx of the PSBT, also a scaladoc describing what you mean by `add` would be nice

Reply:
maybe `addTransactionToInputs` or something along these lines if I understand what is happening here correctly

Reply:
also renaming `i` to `index` would make this easier to understand at first glance